setup/azurecmdfiles.wxi

```diff
@@ -3984,9 +3984,6 @@
                         <Component Id="cmpF7ADD37722F45985B5323202F35958F2" Guid="*">
                             <File Id="fil0FA91D9C31AF07DCD03BFB842FE2B77E" KeyPath="yes" Source="$(var.sourceDir)\ServiceManagement\Azure\Compute\Microsoft.WindowsAzure.Commands.ServiceManagement.PlatformImageRepository.dll" />
                         </Component>
-                        <Component Id="cmp9D64A1111722F25BC100121E73BE039E" Guid="*">
```

Comment:
I tried the build without these changes. But the build started failing saying couldn't find these files. Then I reverted the code to this and build worked.

And I generated it using the exact steps you have given here. I just finally did "git add ., then again build "